src/client/src/services/user-api-service.js

```diff
@@ -11,4 +11,15 @@ export default class UserApiService {
     //***delete when server is ready***
     return user;
   }
+
+  static async updateUser(data) {
+    console.log("update", data);
+    //***uncomment when server is ready***
+    // const response = await fetch(`/users/${data.id}`);
```

Comment:
maybe axios? like in other servises (post answers). so we need proxy and do not need cors. The endpoint is /user. And you pass everything in the body. Description is in readme